src/commands/guess.ts

```diff
@@ -0,0 +1,179 @@
+import dayjs from 'dayjs';
+import { CommandInteraction, MessageActionRow, MessageButton, PermissionString } from 'discord.js';
+import { RESTPostAPIChatInputApplicationCommandsJSONBody } from 'discord-api-types/v10';
+import { Command } from '../interfaces/Command';
+import { t } from '../utils/replyHelper';
+import { isUserInDB, registerUser, getDay, getWord, setPlayed } from '../database';
```

Comment:
I only saw this because I tried to build it (I think the lint workflow should cover this)

```suggestion
import { getUserStatus, registerUser, getDay, getWord, setPlayed } from '../database';
```